todo-MVC-Kuzzle-VueJS/src/main.js

```diff
@@ -0,0 +1,21 @@
+import Vue from 'vue';
+import App from './App.vue';
+import store from './store';
+import router from './router';
+import VueIziToast from 'vue-izitoast';
+import Vuetify from 'vuetify';
+import 'material-design-icons-iconfont/dist/material-design-icons.css';
+import 'vuetify/dist/vuetify.min.css';
+import 'izitoast/dist/css/iziToast.css';
+
+Vue.use(VueIziToast);
+Vue.use(Vuetify, {
+  iconfont: 'md'
+});
+Vue.config.prodctionTip = false;
```

Comment:
Is this a Javascript Setter Typo? :open_mouth:
```suggestion
Vue.config.productionTip = false;
```